test/server/rules.bzl

```diff
@@ -15,30 +15,23 @@
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 #
 
-def grakn_test(grakn_artifact = None,
-               deps = [],
-               classpath_resources = [],
-               data = [],
-               **kwargs):
+def grakn_java_test(name, grakn_artifact_linux, grakn_artifact_mac, deps = [], classpath_resources = [], data = [], **kwargs):
 
-    new_data = [] + data
-
-    if grakn_artifact != None:
-        new_data += [grakn_artifact]
-
-    location = "$(location {})".format(grakn_artifact) if grakn_artifact != None else "none"
+    native.genrule(
+        name = "grakn_artifact_selector",
```

Comment:
we should be using dashes for our own target names - workspaces have underscores